pvactools/lib/load_gtf_data.py

```diff
@@ -37,6 +38,13 @@ def execute(self):
                 (gtf_df_all['feature'].isin(['CDS', 'transcript']))
                 ].replace(["^\s*$"], 'NA', regex=True)
             gtf_df['transcript_biotype'] = 'Not Supported'
+        
+        if 'tag' in gtf_df_all:
+            if self.allow_incomplete_transcripts:
+                gtf_df.loc[~gtf_df['tag'].str.lower().str.contains("cds_start_nf|cds_end_nf", na=False), 'tag'] = "None"
+            else:
+                gtf_df = gtf_df.loc[~gtf_df['tag'].str.lower().str.contains("cds_start_nf|cds_end_nf", na=False)].copy()
+                gtf_df['tag'] = "NA"
 
```

Comment:
This logic doesn't seem to match the logic for the VCF. We also need to decide what to do when the `tag` field is missing.